ember/translations/en.yaml

```diff
@@ -136,6 +141,21 @@ page:
     edit:
       title: "Edit role"
 
+  profile:
+    title: "Profile"
+    switcher:
+      identity: "Identity"
+      memberships: "Memberships"
+    organisation:
+      edit: "Edit organisaton"
+      empty: "No organisations yet."
+
+  notfound:
+    title: "404"
+    subtitle: "Page was not found"
+    link: "Back to"
+    home: "home page"
```

Comment:
Is the `home` intentionally lowercase?

Reply:
yes as it combines with the previous string for "Back to home page"